### DISCUSSION TITLE
Can't send email via smtp with tls

### DISCUSSION THREAD

**Discussion by Mishanki** [laravel/framework#41075] [Q&A]
- Laravel Version: 8.83.1
- PHP Version: 8.1.0
- Database Driver & Version: Pgsql

### Description:

Error message: stream_socket_enable_crypto(): SSL: Success
file: /var/www/vendor/swiftmailer/swiftmailer/lib/classes/Swift/Transport/StreamBuffer.php
line: 94

**Work on localhost (Mac OS) and not working on remote Ubuntu.** 

### Steps To Reproduce:

Call email: Mail::to('test@test.ru')->send(new OrderShipped());

Options:
```php
'smtp' => [
            'transport' => 'smtp',
            'host' => env('MAIL_HOST', 'smtp.mailgun.org'),
            'port' => env('MAIL_PORT', 587),
            'encryption' => env('MAIL_ENCRYPTION', 'tls'),
            'username' => env('MAIL_USERNAME'),
            'password' => env('MAIL_PASSWORD'),
            'timeout' => null,
            'auth_mode' => null,
            'stream' => [
                'ssl' => [
                    'allow_self_signed' => true,
                    'verify_peer' => false,
                    'verify_peer_name' => false,
                ],
       ],
]
```

In .env:
```
MAIL_MAILER=smtp
MAIL_ENCRYPTION=tls
```

**Comment by gguney** [comment]
Did you try to change MAIL_MAILER to MAIL_DRIVER

**Reply by Mishanki** [reply]
Same error

**Comment by fulopattila122** [comment]
Has this ever been resolved?

As far as I see, even after the SwiftMailer => Symfony Mailer replacement, the [option is the same](https://github.com/symfony/mailer/blob/v6.0.8/Transport/Smtp/EsmtpTransportFactory.php#L32)

**Reply by Mishanki** [reply]
Finnally - remote problem with email server.

**Comment by fulopattila122** [comment]
As far as I see, you need to move the stream config options from under the `smtp` key one level higher.

[Referenced from here](https://github.com/laravel/framework/pull/38481):
> Setting stream options for the SMTP transport can no longer be done through an smtp key. Instead you need to set the options directly in the config. For example, when disabling TLS peer verification (see https://github.com/symfony/symfony/pull/35262)

**Comment by zignuts-techno** [comment]
If someone is using AWS SES, then with recent changes they disabled TLS 1.0 and TLS 1.1 versions, our mail sending activities were affected. As a remedy, I changed the port to 465 and encryption to ssl in the env file (MAIL_ENCRYPTION=ssl) fixed the issue for me.

**Reply by maplenk** [reply]
Thank You soooo much @zignuts-techno !!

**Reply by wuaguxtech** [reply]
Muchas Gracias por tu aporte me ayudo mucho
